package.json

```diff
@@ -3,152 +3,103 @@
   "version": "0.17.1",
   "private": true,
   "scripts": {
-    "lint": "eslint --ext .js --ext .jsx .",
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "ios": "node node_modules/.bin/react-native run-ios",
-    "android": "node node_modules/.bin/react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/ && node node_modules/.bin/react-native run-android",
-    "node-hack": "node node_modules/.bin/rn-nodeify --hack",
+    "android": "react-native run-android",
```

Comment:
Is the bundle working correctly after the change in this command?

Reply:
It seems to run as intended on the device and emulator, is there any specific behavior you want tested?

Reply:
The previous `npm run android` command used to create the release bundle.

```
"android": "node node_modules/.bin/react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/ && node node_modules/.bin/react-native run-android",
    "node-hack": "node node_modules/.bin/rn-nodeify --hack",
    "android": "react-native run-android",

```

I'm wondering if this still works.

Reply:
The old command looks weird to me

Do you remember if there was any specific reason for having it like that? It seems to me that `react-native run-android` should already create the bundle and start the development environment